dime-research-standards/pillar-4-data-security/data-security-resources/surveycto-encryption-guidelines.md

```diff
@@ -0,0 +1,274 @@
+In this case you decide together with the survey firm
+which is the minimum list of fields that they need to track progress.
+If all of these variables are _not_ sensitive (which often is the case)
+then you can make all of them publishable.
+Then you can give the survey firm permission to download data from the server
+or view it in the Data Explorer,
+but as long as they do not have access to the private key,
+they have no access to fields that not publishable.
+
+To make a field publishable, you simply write "_yes_" in the _publishable_ column in the _survey_ tab in the questionnaire form.
+
+To download the publishable data, go to the _Export_ tab in your SurveyCTO server.
+Find your form and click "_Download for data_".
+Then in the two sections as normal,
+but in the "_Fields to include_" make sure that the checkbox
+"_Publishable fields only (if you don't have the private key)_" is checked.
+Then click _Download .csv now_.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-publish-1.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+## Topic 4 - Using your private key to decrypt your data in SurveyCTO
+
+Technically you can download encrypted data from your SurveyCTO server using a browser,
+but then you first need to save your private key in a file on your computer.
+In the workflow we recommend,
+you really want to avoid to ever store the private key in a file on your computer
+after you have saved it in a password manager.
+Instead, we recommend you to only download data using [SurveyCTO Desktop](https://docs.surveycto.com/desktop/)
+as you can copy your key from your password manager and paste it
+without having to save it in a file first.
+
+In SurveyCTO Desktop you log in to your server,
+and click _Sync_ in the menu to the left.
+Fill in the information as normal,
+and in step 2 you need to paste the private key. See image below.
+Go to your password manager and copy your private key.
+Remember to include the header `-----BEGIN RSA PRIVATE KEY-----`
+and the footer `-----END RSA PRIVATE KEY-----`.
+Once you have copied it to your clipboard
+(ctrl-C on Windows and command-C on Mac)
+then simply click the _PASTE KEY_ button in SurveyCTO Desktop.
+If the key you pasted is on the excepted format,
+then you will see a green checkmark
+and the text "_Private key successfully pasted_".
+However, SurveyCTO has not tested if you pasted the correct key,
+it will only test that when you are actually downloading data.
+If you have pasted the incorrect key, then you will get an error
+and you will not be able to read the data.
+
+<img src="https://github.com/worldbank/dime-standards/blob/scto-guidelines/dime-research-standards/pillar-4-data-security/data-security-resources/img/scto-sync-1.png" width="75%"><!--- Image is read from master branch or use full URL-->
+
+## Topic 5 - Testing your setup before
```

Comment:
before what???? ;-)

Reply:
done in efac4632a617c269d6083b8a7ffa1a2ff104ea00

Reply:
done in efac4632a617c269d6083b8a7ffa1a2ff104ea00